tools/wrangling/.shed.yml

```
@@ -0,0 +1,8 @@
+categories:
+  - Text Manipulation
+description: Convert an xlsx file to a tabular
+long_description: Extract one sheet from an xlsx/xls file and convert it to a tabular format
+name: excel_to_tabuler
+owner: ufz
+homepage_url: https://github.com/Helmholtz-UFZ/galaxy-tools
+remote_repository_url: https://github.com/bernt-matthias/mb-galaxy-tools/tools/tox_tools/wrangling
```

I would suggest to rename the folder to xls2tsv, or?

```suggestion
remote_repository_url:https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/xls2tsv
```